build/linux/Makefile.in

```diff
@@ -281,13 +281,13 @@ ifeq ($(python),yes)
 py: libastra.la
 	$(MKDIR) python/build
 	$(MKDIR) python/finalbuild
-	cd $(srcdir)/../../python; CPPFLAGS="${PYCPPFLAGS}" LDFLAGS="${PYLDFLAGS}" $(PYTHON) builder.py build --build-base=$(abs_top_builddir)/python/build install \
+	cd $(srcdir)/../../python; CXX="${CXX}" CC="${CXX}" CPPFLAGS="${PYCPPFLAGS}" LDFLAGS="${PYLDFLAGS}" $(PYTHON) builder.py build --build-base=$(abs_top_builddir)/python/build install \
```

Comment:
I could not find CC being set by the `Makefile.in` script, so I figured to set it to CXX as an alternative. If I don't set it, it seems like the python compilation script does the compilation with the system default `gcc` binary. This doesn't really make sense to me but the result is that CC should at least be set to something. 

I don't really know how this whole system works though, so if you have a better solution, I'm fine with that.